fpdf/table.py

```diff
@@ -282,7 +303,7 @@ def get_cell_border(self, i, j):
             if j == columns_count - 1:
                 border.remove("R")
         if self._borders_layout == TableBordersLayout.NO_HORIZONTAL_LINES:
```

Comment:
Could you please use this layout in a unit test with `num_heading_rows > 1` , so that we have a non-regression test checking the expected behaviour in that case